deck.md

```diff
@@ -1366,6 +1366,125 @@ export default async function users(fastify) {
 
 ---
 
+class: branded
+
+# Step 13: Exercise 💻
+
+- Let's create an Fastify application using **Typescript**.
```

Comment:
Is it worth ito mention eslint for typescript? My code editor was freaking out on the .ts files until I added ```@typescript-eslint/parser``` and ```@typescript-eslint/eslint-plugin```. Could be confusing for beginners.

Reply:
nitpick: it's `TypeScript`, not `Typescript`, right?

Reply:
Done!